src/formatOnSave/isPrettierInPackageJson.js

```diff
@@ -35,7 +35,7 @@ const isPrettierInPackageJson: (editor: TextEditor) => boolean = _.flow(
 const isPrettierEslintInPackageJson: (editor: TextEditor) => boolean = _.flow(
   readContentsOfNearestPackageJson,
   _.overSome([
-    hasPackage('prettier-eslint'),
+    hasPackage('@lewisl9029/prettier-eslint'),
```

Comment:
This is subtle but this code here is trying to determine whether to enable prettier-atom format on save or not based on whether or not the user already has prettier, or in this case, prettier-eslint, in their package-json. Therefore, I think the change we need here is to actually cue on _either_ package being present (in the off change they do have your package installed). Either that, or revert this change?

Reply:
Ah great catch. Reverting sounds like the way to go. I don't expect my fork to get installed in projects outside of this one. 😅 

Since this repo itself looks like it might be dogfooding this extension, I'll also add prettier-eslint to devDependencies so we don't need any special logic to handle that.

Reply:
Pushed up the changes. Let me know if you see anything else weird.